incubator/squid/templates/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "squid"
+  labels:
+    app: squid
```

Comment:
please use the new labels, see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata